src/flat-button.jsx

```diff
@@ -197,7 +258,7 @@ const FlatButton = React.createClass({
         onKeyboardFocus={this._handleKeyboardFocus}
         onMouseLeave={this._handleMouseLeave}
         onMouseEnter={this._handleMouseEnter}
-        onTouchStart={this._handleTouchStart}
```

Comment:
> I will leave that prop there for now.

As you want, but we have to go to the bottom of this. Either way, we need to be coherent.